src/main/plugin/iso19139.ca.HNAP/layout/utility-tpl-multilingual.xsl

```diff
@@ -90,8 +96,14 @@
                   <xsl:otherwise><xsl:value-of select="$mainLanguage"/></xsl:otherwise>
                </xsl:choose>
             </xsl:variable>
+            <xsl:variable name="mainLanguageId_ISO639_2B">
```

Comment:
Changed the value of an `@id` reference/variable name seems incorrect as this is only a reference to the PT_Locale.  It could be called `ABC`  for referencing the English PT_Locale.